mithril-test-lab/mithril-aggregator-fake/src/error.rs

```diff
@@ -3,15 +3,15 @@ use axum::{
     http::{Response, StatusCode},
     response::IntoResponse,
 };
-use tracing::{debug, error};
+use tracing::error;
 
 /// error that wraps `anyhow::Error`.
 #[derive(Debug)]
 pub enum AppError {
     /// Catching anyhow errors
     Internal(anyhow::Error),
 
-    /// Resource not found (specify what)
+    /// Resource not found (specify body)
     NotFound(String),
```

Comment:
What utility does the String have if we only set it as an empty string ? You should remove it altogether.

Reply:
We've left it there in case we need it in the future. >e removed it to clean up the tests 👍.